miniapps/nurbs/nurbs_curveint.cpp

```diff
@@ -0,0 +1,238 @@
+// Compile with: make nurbs_curveint
+//
+// Sample runs:  ./nurbs_curveint -uw -n 9
+//               ./nurbs_curveint -nw -n 9
+
+//
+// Description:  This example code demonstrates the use of MFEM to interpolate
+//               a curve in a NURBS patch. We first define a square shaped
+//               NURBS patch. We then interpolate a sine function on the bottom
+//               edge. The results can be viewed in VisIt.
+//
+//               Us use curve interpolation for curves with all weights being 1,
+//               B-splines, and curves with not all weights being 1, NURBS. The
+//               spacing in both cases is chosen differently.
+
+
+
+#include "mfem.hpp"
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+using namespace mfem;
+
+KnotVector *UniformKnotVector(int order, int ncp)
+{
+   if (order>=ncp)
+   {
+      mfem_error("UniformKnotVector: ncp should be at least order + 1");
+   }
+   KnotVector *kv = new KnotVector(order, ncp);
+
+   for (int i = 0; i < order+1; i++)
+   {
+      (*kv)[i] = 0.0;
+   }
+   for (int i = order+1; i < ncp; i++)
+   {
+      (*kv)[i] = (i-order)/double(ncp-order);
+   }
+   for (int i = ncp ; i < ncp + order + 1; i++)
+   {
+      (*kv)[i] = 1.0;
+   }
+   return kv;
+}
+
+
+
+int main(int argc, char *argv[])
+{
+   // Parse command-line options.
+   OptionsParser args(argc, argv);
+
+   double l       = 1.0;
+   double a       = 0.1;
+   int ncp        = 9;
+   int order      = 2;
+   bool ifbspline = true;
+
+   args.AddOption(&l, "-l", "--box-side-length",
+                  "Height and width of the box");
+   args.AddOption(&a, "-a", "--sine-ampl",
+                  "Amplitude of the fitted sine function.");
+   args.AddOption(&a, "-a", "--sine-ampl",
```

Comment:
The `-a` option is added twice.

Reply:
Resolved in [e31bfca](https://github.com/mfem/mfem/pull/3232/commits/e31bfca43bf510dfc9e0cd51398ea06efa9707c6)